src/palace/manager/api/controller/loan.py

```diff
@@ -467,39 +504,39 @@ def revoke(self, license_pool_id):
 
         work = pool.work
         annotator = self.manager.annotator(None)
+        single_entry_feed = OPDSAcquisitionFeed.single_entry(work, annotator)
+        if single_entry_feed is None:
+            return INTERNAL_SERVER_ERROR
         return OPDSAcquisitionFeed.entry_as_response(
-            OPDSAcquisitionFeed.single_entry(work, annotator),
+            single_entry_feed,
             mime_types=flask.request.accept_mimetypes,
         )
 
-    def detail(self, identifier_type, identifier):
-        if flask.request.method == "DELETE":
-            return self.revoke_loan_or_hold(identifier_type, identifier)
```

Comment:
I removed the `DELETE` endpoint here, because `revoke_loan_or_hold` is undefined. Looking through our logs, it seems that this endpoint never gets called with `DELETE`, so it doesn't seem worth it to fix.